tests/framework/CodeInterfaceTests/CobraTF/CobraTFinterfaceTest4/testDummyStep_MC/1/heat_exchanger.inp

```diff
@@ -0,0 +1,220 @@
+** Author: Bob Salko
+** Date: 6/15/17
```

Comment:
Seems to me, you do not need this file. This file will generated by raven, right?